requirements/test.txt

```diff
@@ -5,7 +5,6 @@
 #    pip-compile --output-file=test.txt test.in
 #
 agentarchives==0.4.0
-appnope==0.1.0            # via ipython
```

Comment:
(This deleted itself running the requirements `Makefile`)

Reply:
Argh, I was wondering when this was added. It looks like...ipython requires appnope on os x. See here: https://github.com/ipython/ipython/blob/master/setup.py#L204